.github/workflows/commit-queue.yml

```diff
@@ -32,6 +32,7 @@ jobs:
     steps:
       - name: Get Pull Requests
         id: get_mergeable_prs
+        # TODO: the filtering below should also check for the correct number of approvals
```

Comment:
I'm also piggybacking this TODO in this PR, as I imagine that we'd want the correct number of approvals to also be taken into consideration? (or I'm mistaken and it's considered already?)

Reply:
The CQ already rejects a PR that have not enough approvals if that’s what you are asking

Reply:
thanks :slightly_smiling_face: yeah that's what I'm asking :slightly_smiling_face: 

Just for my understanding could you share where the check is being made? because I am failing to see it in either this yml file or the [CQ script](https://github.com/nodejs/node/blob/7112fb9fd7281917c268959bfb25e8f12f41218e/tools/actions/commit-queue.sh) :pray:

Reply:
It's in https://github.com/aduh95/node-core-utils/blob/a88776bdca7555798fab9f1ee65bfa3adecdb1d5/lib/pr_checker.js#L132-L233

Reply:
oh, I see!

makes sense, thanks a lot! :heart_hands:

Reply:
comment removed :slightly_smiling_face: